sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormLine.java

```diff
@@ -82,7 +82,7 @@ public int getPageNumber() {
      *
      * @param appearance the appearance text line.
      */
-    private FormLine setAppearance(Appearance appearance) {
```

Comment:
I think appearance is missing in the docstring for FormLine

Reply:
Do you mean [here](https://github.com/Azure/azure-sdk-for-java/pull/18823/files/262ab3e9b6102637147e683f272ad8da23ab995f#diff-0d7faae38c70ad7d20f96b04ac68a0c5654f1178184609eaff10614b2c4e7464L21). Since it is a private property this can be ignored.